exchanges/kucoin/kucoin_websocket.go

```diff
@@ -1272,8 +1247,8 @@ type update struct {
 // job defines a synchronisation job that tells a go routine to fetch an
 // orderbook via the REST protocol
 type job struct {
-	Pair      currency.Pair
-	AssetType asset.Item
+	Pair currency.Pair
+	// AssetType asset.Item
```

Comment:
[DEBUG] | SYNC | 08/03/2023 12:00:51 | Kucoin orderbook sync complete UNI-USDTM [4/6].

Does futures, and spot but loses margin for websocket.